src/util/widgetrendertimer.cpp

```diff
@@ -22,8 +24,16 @@ void WidgetRenderTimer::guiTick() {
 }
 
 void WidgetRenderTimer::activity() {
-    m_lastActivity = mixxx::Time::elapsed();
-    if (!m_guiTickTimer.isActive()) {
-        m_guiTickTimer.start(m_renderFrequency);
+#ifdef __APPLE__
```

Comment:
This requires some comments.

Reply:
Is https://github.com/mixxxdj/mixxx/blob/23b851befd556d7178de5b6ade57ff73aed4e57e/src/waveform/waveformwidgetfactory.cpp#L692
not reached? 

I think it would be better to repair the instead of clutter every Widget with is workaround. 
If this is not possible i think it would be cleaner to indicate by [Master]", "guiTickTime" = -1 that it is not working. 
Than every user of this CO can check this easily.

Reply:
The comments are in the header already.